doc/man1/dgst.pod

```diff
@@ -2,13 +2,14 @@
 
 =head1 NAME
 
-dgst, sha, sha1, mdc2, ripemd160, sha224, sha256, sha384, sha512, md4, md5, blake2b, blake2s - message digests
+dgst
+- generate or verify message digests
 
 =head1 SYNOPSIS
 
-B<openssl> B<dgst>
+B<openssl dgst>
+[B<-[digest]>]
```

Comment:
Originally question also was what does `[-[digest]]` mean? That passing dash alone is meaningful? I mean shouldn't it be at least `[-digest]`. And by "at least" I mean that I see nothing wrong with original `[B<-I<digest>>]`...